identity/src/keypair.rs

```diff
@@ -773,7 +854,13 @@ mod tests {
     }
 
     #[test]
-    #[cfg(feature = "peerid")]
+    #[cfg(all(
```

Comment:
Here initially the `keypair_from_protobuf_encoding` test failed due to its `feature = "peerid"` property that is not taken into account by the `from_protobuf_encoding` function and recursively for encapsulated functions such as `bad_protobuf` or `decoding_unsupported`. As a result, I opened new features here to secure the test while not changing the core codebase.

Reply:
Makes sense, thank you!